packages/venia-ui/lib/components/CartPage/PriceSummary/priceSummary.css

```diff
@@ -26,6 +26,12 @@
     font-weight: bold;
 }
 
+.priceUpdating {
+    @composes: price;
+    color: white;
+    background: white;
```

Comment:
This just covers the "stale" price data. Should I use a different method to cause this effect?

Reply:
Preferable to use `opacity` here. I also saw some older XD mocks that had a transition which I think we should add. This seems to work, but causes an odd cascade; review with UX and if its too much trouble we can leave as-is.
```css
.price {
    justify-self: end;
    transition: opacity 384ms var(--venia-anim-standard);
    opacity: 1;
}

.priceUpdating {
    composes: price;
    opacity: 0;
}
```